package.json

```diff
@@ -5,7 +5,7 @@
     "An API to help facilitate the use of the Yahoo! Fantasy Sports API in NodeJS projects.",
   "main": "index.js",
   "scripts": {
-    "test": "node -r @std/esm jasmine-runner.js",
+    "test": "node -r esm jasmine-runner.js",
```

Comment:
Scoped package `@std/esm` is not currently in the package.json, but plain `esm` is. Could alternatively switch the package.json to  `@std/esm`.